repo_policy_compliance/__init__.py

```diff
@@ -115,6 +136,31 @@ def pull_request(
     return check.Report(result=check.Result.PASS, reason=None)
 
 
+def _retrieve_policy_document(
```

Comment:
minor: according to the step-down rule this fct should be placed at the bottom of the file